src/index.html

```diff
@@ -7,8 +7,49 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a href="#home">
+        <img 
+        src="./images/logo.png" 
+        alt="icon Moyo" 
+        class="logo_Moyo"
+        >
+      </a>
+      <nav class="navigation">
+        <ul class="nav_list">
+          <li class="nav_item"><a href="#" class="is-active">APPLE</a></li>
+
+          <li class="nav_item"><a href="#" class="nav_link">SAMSUNG</a></li>
```

Comment:
if it is one line element, don't add blanks between sibling

if it is multiline elements, you should add it

now you made it like one line, so no need to to it
```suggestion
          <li class="nav_item"><a href="#" class="is-active">APPLE</a></li>
          <li class="nav_item"><a href="#" class="nav_link">SAMSUNG</a></li>
```